common.rb

```diff
@@ -90,7 +98,7 @@ def put_command(cmd, redact=nil)
   end
 
   def capture_stdout(cmd)
-    output, _ = Open3.capture2(*cmd, :err=>"/dev/null")
+    output, _ = Open3.capture2(*cmd)
```

Comment:
I should have put a comment here, but I believe I found it distracting to see STDERR logs when I was just trying to capture STDOUT of a command. Instead, I think it should be an argument with a default of not suppressing STDERR.

Reply:
Sounds good, done.

I don't plan to update any call sites to use the parameter since I don't know where they would be, but hopefully we could do so going forward.

For context, this stderr suppression made debugging very difficult when implementing our initial AoU deployment scripts. Of course during my initial attempts, the underlying command lines were failing.